modules/rostests/apitests/shell32/SHLimitInputEdit.cpp

```diff
@@ -238,17 +238,10 @@ PropEnumProc(
     return TRUE;
 }
 
-static INT_PTR CALLBACK
-DialogProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
+static void
+Test1(HWND hEdt1)
```

Comment:
Any chance to name these functions by what they test instead?

Reply:
OK.